2003-07-29  Jakub Jelinek  <jakub@redhat.com>

	* ld-elfvsb/elf-offset.ld: Add .rel.toc, .rela.toc and .toc
	sections.

--- ld/testsuite/ld-elfvsb/elf-offset.ld.jj	2003-05-19 17:12:47.000000000 -0400
+++ ld/testsuite/ld-elfvsb/elf-offset.ld	2003-07-29 05:39:34.000000000 -0400
@@ -46,6 +46,8 @@ SECTIONS
     }
   .rel.got       : { *(.rel.got)		}
   .rela.got      : { *(.rela.got)		}
+  .rel.toc       : { *(.rel.toc)		}
+  .rela.toc      : { *(.rela.toc)		}
   .rel.ctors     : { *(.rel.ctors)	}
   .rela.ctors    : { *(.rela.ctors)	}
   .rel.dtors     : { *(.rel.dtors)	}
@@ -111,6 +113,7 @@ SECTIONS
     KEEP (*(.dtors))
   }
   .got           : { *(.got.plt) *(.got) }
+  .toc           : { *(.toc) }
   .dynamic       : { *(.dynamic) }
   /* We want the small data sections together, so single-instruction offsets
      can access them all, and initialized data all before uninitialized, so
